op_builder/fused_lamb.py

```diff
@@ -4,6 +4,15 @@
 import torch
 from .builder import CUDAOpBuilder, is_rocm_pytorch
 
+if is_rocm_pytorch:
+    with open('/opt/rocm/.info/version-dev', 'r') as file:
+        ROCM_VERSION_DEV_RAW = file.read()
+    ROCM_MAJOR = (ROCM_VERSION_DEV_RAW.split('.')[0])
+    ROCM_MINOR = (ROCM_VERSION_DEV_RAW.split('.')[1])
+else:
+    ROCM_MAJOR = '0'
+    ROCM_MINOR = '0'
+
```

Comment:
This should instead be something like: `from .builder import ROCM_MAJOR, ROCM_MINOR`. We shouldn't need to redefine it here.